src/amber/router/params_parser.cr

```diff
@@ -78,13 +78,18 @@ module Amber::Router
     end
 
     def merge_route_params
-      route_params.each do |k, v|
-        if k == route_params.keys.last
-          params[k] = v.sub(Controller::Base::Content::REGEX, "")
-        else
-          params[k] = v
-        end
+      route_params_without_ext.each do |k, v|
+        params[k] = v
+      end
+    end
+
+    def route_params_without_ext
+      rparams = route.params
```

Comment:
you can use `_params`or `route_params` instead of `rparams`, the latter can be confusing. 

Thanks for splitting it into 2 separate methods.

Reply:
I just wanted something short and temporary what would overwrite anything else. To me rparams looked better than _params. WDYT?